### PR TITLE
OTJ: TriggerBecomesPlotted (and other Plot related stuff)

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Longhorn Sharpshooter
+ManaCost:2 R
+Types:Creature Minotaur Rogue
+PT:3/3
+K:Reach
+T:Mode$ BecomesPlotted | ValidCard$ Card.Self | TriggerZone$ Battlefield | Execute$ TrigDealDamage | TriggerDescription$ When CARDNAME becomes plotted, it deals 2 damage to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt:6]: wrong TriggerZone

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt]: your params still have a Typo
gotta test how this will play out

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt]: The trigger was still "working" with either Zone... though I did notice one other glitch I was going to ask about.
Testing with right param now to see if the glitch gets solved

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt]: Weird glitches:
 - if a trigger goes off, the plotted card stays face up (don't know if this is that big a deal as I'm not sure plotted cards should be turned face-down anyway?)
 - the trigger also seems to create a second "play for free" ability?
 
image

**Comment by Hanmac** [comment]
While you are at this, can you remove that Plot exiles face down?

File: forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/longhorn_sharpshooter.txt]: just get rid of the facedown part

File: forge-gui/res/cardsfolder/upcoming/doc_aurlock_grizzled_genius.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Doc Aurlock, Grizzled Genius
+ManaCost:G U
+Types:Legendary Creature Bear Druid
+PT:2/3
+S:Mode$ ReduceCost | ValidCard$ Card.wasCastFromYourGraveyard,Card.wasCastFromExile | Type$ Spell | Activator$ You | Amount$ 2 | Description$ Spells you cast from your graveyard or from exile cost {2} less to cast.
+S:Mode$ ReduceCost | ValidSpell$ Static.Plotting | Activator$ You | Amount$ 2 | Description$ Plotting cards from your hand costs {2} less.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/doc_aurlock_grizzled_genius.txt:6]: hand check missing

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/doc_aurlock_grizzled_genius.txt]: fixed - checked it with that Fblthp to make sure!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**